src/node/in_memory.rs

```diff
@@ -1242,6 +1220,7 @@ impl<S: ForkSource + std::fmt::Debug + Clone> InMemoryNode<S> {
                 "  Slot overhead:    {:>15}",
                 to_human_size(bootloader_debug.overhead_for_slot)
             );
+            tracing::info!("Also, your transaction has contributed by execution and pubdata, but it's included in the gas and pubdata price");
```

Comment:
The verbiage is confusing for this message. Could you expand on what you're trying to communicate and perhaps there's some missing words that we can add to clarify the message.

Reply:
Yes, I added this comment cause removed some data. But, probably, it makes sense to add information according to the new overhead system and definitely make this comment clear